server/controllers/bookController.js

```diff
@@ -116,6 +117,19 @@ class bookController {
       });
     }
   }
+
+  static async viewAvailableBooks(req, res) {
```

Comment:
Please probably 
Use the following if to check to admin 
For the isLogedIn returned res.user

if (req.user.isAdmin !== true) {
        return res.status(401).json({
          status: 401,
          error: 'You are not allowed to perform this action',
        });
      }

Reply:
Also, the status code should be forbidden